tests/CoalaBearTest.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import unittest
+
+import bears
+import coalib
+
+
+class CoalaBearTest(unittest.TestCase):
+
+    def test_check_coala_version(self):
+        bears.__version__ = '1.0.0.999999999'
+        coalib.__version__ = bears.__version__
```

Comment:
And you can do `coalib.__version__ = bears.__version__ = '1.0.0.dev999999999'` to make it more clear